### PR TITLE
Add rt-utils

### PR DESCRIPTION
**Pull request by ramtingh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ramtingh** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/rt-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "rt-utils" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qurit/rt-utils/archive/6c52b8d10934907ad36250d1e01894adbe5ec86f.zip
```
Comment (ocefpaf) [inline recipes/rt-utils/meta.yaml:9]: Do they have a tag that reflects the version 1.2.1? Using a commit makes reduces transparency and will break the updating bot.

Comment (ramtingh) [reply recipes/rt-utils/meta.yaml]: @ocefpaf yes, I've switched it over to using the tag

File: recipes/rt-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qurit/rt-utils/archive/refs/tags/v{{ version }}.zip
+  sha256: 547be04ddaa566c357bea802392576aa3adf4d19f9284e4120ff21ebc308ec6b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps . -vv
```
Comment (ocefpaf) [inline recipes/rt-utils/meta.yaml:14]: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Pull request closed without merging**